recipes/juliacall/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.4.3" %}
+
+package:
+  name: juliacall
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/juliacall/juliacall-{{ version }}.tar.gz
+  sha256: ff81c152831775bbbb26b7752a7825d1de2fd6c154f71ffc22338996b4faac07
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
This is a pure Python package, it should be `noarch: python`. Remove the skip and set a minimum version on Python in host and run `>=3.6` instead.